lib/job.jsonnet

```diff
@@ -36,6 +36,7 @@ function(jobName, agentEnv={}, stepEnvFile='', patchFunc=identity) patchFunc({
     BUILDKITE_PLUGIN_K8S_BUILD_PATH_PVC: '',
     BUILDKITE_PLUGIN_K8S_GIT_MIRRORS_HOST_PATH: '',
     BUILDKITE_PLUGIN_K8S_MOUNT_SECRET: '',
+    BUILDKITE_PLUGIN_K8S_MOUNT_SECRET_PERMISSIONS: '256',
```

Comment:
Write out that this is 400 octal in a comment, it's easier to think about permissions in oct and I'm not great at translating dec -> oct in my head :)